pkg/cli/admin/upgrade/status/examples/4.16.0-ec2-control-plane-updated-pdb-prohibits-draining.detailed-output

```diff
@@ -1,15 +1,8 @@
 = Control Plane =
-Assessment:      Completed
-Target Version:  4.16.0-ec.3 (from 4.16.0-ec.2)
-Completion:      100% (36 operators updated, 0 updating, 0 waiting)
-Duration:        3h31m
-Operator Status: 35 Healthy, 1 Available but degraded
+Cluster version is 4.16.0-ec.3
```

Comment:
I think we want to use a language that says "complete! done! all is good!" more explicitly :D Also potentially still provide more information?

"Update to $version successfully completed at $when (duration: $duration)"